documentation.js

```diff
@@ -0,0 +1,23 @@
+const dictum = require('dictum.js');
```

Comment:
This file is unnecessary. When you call `dictum.chai(response)` the file with documentation is created automatically.